### DISCUSSION TITLE
Headless Installer for RabbitMq server

### DISCUSSION THREAD

**Discussion by dssaggu1991** [rabbitmq/rabbitmq-jms-client#388] [Q&A] [answered]
I wonder if I can install RabbitMQ server on windows using Command without any GUI ?

**Comment by michaelklishin** [comment, marked as answer]
How does this have anything to do with this client library?

[Chocolatey](https://rabbitmq.com/install-windows.html#chocolatey) is the best option I know for CLI-driven installation of RabbitMQ on Windows.

**Comment by lukebakken** [comment]
@dssaggu1991 yes you can. The windows installer uses NSIS - 

https://documentation.help/NSIS/Section4.12.html#:~:text=To%20make%20sure%20your%20installer,default%20answer%20for%20silent%20installers.

In the future, you should ask questions here:

https://github.com/rabbitmq/rabbitmq-server/discussions

**Reply by dssaggu1991** [reply]
Thanks for help
